packages/cli/src/commands/governance/propose.ts

```diff
@@ -44,20 +55,29 @@ export default class Propose extends BaseCommand {
   }
 
   static examples = [
-    'propose --jsonTransactions ./transactions.json --deposit 10000 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631 --descriptionURL https://gist.github.com/yorhodes/46430eacb8ed2f73f7bf79bef9d58a33',
+    'propose --jsonTransactions ./transactions.json --deposit 10000e18 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631 --descriptionURL https://gist.github.com/yorhodes/46430eacb8ed2f73f7bf79bef9d58a33',
+    'propose --jsonTransactions ./transactions.json --deposit 10000e18 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631  --useMultiSig --for 0x6c3dDFB1A9e73B5F49eDD46624F4954Bf66CAe93 --descriptionURL https://gist.github.com/yorhodes/46430eacb8ed2f73f7bf79bef9d58a33',
   ]
 
   async run() {
     const kit = await this.getKit()
     const res = await this.parse(Propose)
     const account = res.flags.from
-    const deposit = new BigNumber(res.flags.deposit)
     kit.defaultAccount = account
-
-    await newCheckBuilder(this, account)
-      .hasEnoughCelo(account, deposit)
-      .exceedsProposalMinDeposit(deposit)
-      .runChecks()
+    const deposit = new BigNumber(res.flags.deposit)
+    if (res.flags.useMultiSig && !res.flags.for) {
+      this.error(
+        'If the --useMultiSig flag is set, then the --for flag has to also be set to an address.'
```

Comment:
nit:  `--for flag has to also be set to an address for the Multisig`

Reply:
I updated the error message at line 66. 

https://github.com/celo-org/developer-tooling/blob/a20e707e9af56ff03e95e4a8f2fd64b6bae06065/packages/cli/src/commands/governance/propose.ts#L66

I assumed your comment is about line 66, and not line 70 

https://github.com/celo-org/developer-tooling/blob/a20e707e9af56ff03e95e4a8f2fd64b6bae06065/packages/cli/src/commands/governance/propose.ts#L71

But let me know if I misunderstood, happy to change the error message in other places.